### DISCUSSION TITLE
assimpjs first beta is released

### DISCUSSION THREAD

**Discussion by kovacsv** [assimp/assimp#4059] [Show and tell]
Hi All,

The first beta version of assimpjs is released just now. This is the emscripten compiled version of the assimp library that runs entirely in the browser. It can import almost all supported file formats, and returns the assimp json as a result.

- GitHub repository: https://github.com/kovacsv/assimpjs
- Live demo site: https://kovacsv.github.io/assimpjs

I'm happy to hear your feedback and ideas.

**Comment by kimkulling** [comment]
Cool! Thanks a lot for sharing this information.
